### ISSUE TITLE
Command xcodebuild failed with exit code 65

### ISSUE THREAD

**Issue by erjdriver** [NativeScript/nativescript-cli#5738]
### Issue Description

I've got a NS 6.x project and the ios build fails with `Command xcodebuild failed with exit code 65`.

In the log below the error message is 4 lines from the bottom.

XCode 14.2

### Relevant log output (if applicable)

```shell
Project successfully prepared (ios)
Building project...
Xcode build...
note: Using codesigning identity override: 
note: Building targets in dependency order
/Users/mini/Projects/my-mobile/platforms/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'IQKeyboardManager' from project 'Pods')
/Users/mini/Projects/my-mobile/platforms/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'Toast' from project 'Pods')
/Users/mini/Projects/my-mobile/platforms/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'Pods-mymobile' from project 'Pods')
/Users/mini/Projects/my-mobile/platforms/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'MDFInternationalization' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'MDFInternationalization' from project 'Pods')
/Users/mini/Projects/my-mobile/platforms/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'MaterialComponents' from project 'Pods')
/Users/mini/Projects/my-mobile/platforms/ios/mymobile.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'mymobile' from project 'mymobile')
warning: Run script build phase 'NativeScript PreLink' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'mymobile' from project 'mymobile')
error: Unable to load contents of file list: '/Users/mini/Projects/my-mobile/platforms/ios/newInputFile.xcfilelist' (in target 'mymobile' from project 'mymobile')
warning: Run script build phase 'NativeScript PostBuild' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'mymobile' from project 'mymobile')
** BUILD FAILED **

Command xcodebuild failed with exit code 65
```

### Environment

tns info
You are using the deprecated nsconfig.json file. Just be aware that NativeScript now has an improved nativescript.config.(js|ts) file for when you're ready to upgrade this project.
✔ Getting NativeScript components versions information...
✔ Component nativescript has 8.5.3 version and is up to date.
✔ Component tns-core-modules has 6.5.27 version and is up to date.
⚠ Update available for component tns-ios. Your current version is 6.5.5 and the latest available version is 6.5.6.
✔ Component tns-android has 6.5.3 version and is up to date.

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/nativescript-cli/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by erjdriver** [comment]
I went into xcode 14 and in Build Phases deleted the Input File Lists and now get

error: Build input file cannot be found: '/Users/mini/Projects/my-mobile/platforms/ios/build/Debug-iphonesimulator/metadata-x86_64.bin'

I've got the output set to 

$(DERIVED_FILE_DIR)/metadata-x86_64.bin

**Comment by rigor789** [comment]
That i believe is solved by this update: 

> Update available for component tns-ios. Your current version is 6.5.5 and the latest available version is 6.5.6.

**Comment by erjdriver** [comment]
Spent way too many hours on this. Thanks.
